src/pvq_encoder.c

```diff
@@ -551,6 +560,11 @@ static int pvq_theta(od_coeff *out, const od_coeff *x0, const od_coeff *r0,
       od_val32 qcg;
       qcg = OD_SHL(i, OD_CGAIN_SHIFT);
       k = od_pvq_compute_k(qcg, -1, -1, 1, n, beta, robust || is_keyframe);
+      {
```

Comment:
Instead of putting a block, I would add the same comments that you added in the ref case